recipes-support/simple-http-server/files/simple-http-server.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pushd /etc/hosted_files
+/usr/bin/python -m SimpleHTTPServer
```

Comment:
Since we are already writing a wrapper script, it would be better to make it a python script that instantiates the SimpleHTTPServer rather than exec'ing again.